### PR TITLE
update erlang to 22.1

### PR DESCRIPTION
**Pull request by benoitc**
#### Description

update erlang port to latest version.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on
macOS 10.15 19A583
Xcode 11.1 11A1027

### CODE REVIEW

File: lang/erlang/Portfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ PortGroup           wxWidgets 1.0
 PortGroup           compiler_blacklist_versions 1.0
 
 name                erlang
-version             21.1
+version             22.1
 revision            1
```
Comment (mf2k) [inline lang/erlang/Portfile]: The `revision` line needs to be reset to 0 when upgrading the `version`.

**Comment by mf2k** [comment]
Please squash the commits.

**Comment by benoitc** [comment]
> Please squash the commits.

done, thought it would be done on merge, sorry for that.

**Comment by cjones051073** [comment]
> > Please squash the commits.
> 
> done, thought it would be done on merge, sorry for that.

We can indeed do it once accepted, but its preferred if the author does it themselves.

**Pull request merged**